.github/workflows/publish_tiber.yaml

```diff
@@ -0,0 +1,73 @@
+name: dockerfiles-publish
+
+permissions:
+  contents: read
+
+on:
+  push:
+    branches:
+    - 'main'
+    - 'release-*'
+    tags:
+    - 'v*'
+
+env:
+  BUILD_TYPE: "Release"
+  DOCKER_IMAGE_NAME: "ghcr.io/openvisualcloud/intel-tiber-broadcast-suite/tiber-broadcast-suite"
```

Comment:
are we allowed to use ghrc.io ?
it's internal github registry

Reply:
This approach is the only solution as viewed from Docker and docker-actions know-how point of view. I am not yet fully utilizing it as one should but this must be done step by step.

Reply:
at this moment, I suggest to disable this action, code can stay here, but do not execute it yet

Reply:
The push action is disbaled. Caching stays at current minimal level - as same as in the MCM